.github/workflows/android-build.yml

```diff
@@ -80,5 +89,4 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: Status-tablet
-          path: mobile/bin/android/qt6/Status-tablet.apk
-
+          path: mobile/bin/android/qt6/**/Status-tablet.apk
```

Comment:
Should not be needed. We have an open PR that's fixing this. We don't want to release by accident an app version with qml server enabled.